pkg/e2e/infra/infra.go

```diff
@@ -485,22 +316,10 @@ var _ = g.Describe("[Feature:Machines] Managed cluster should", func() {
 		g.By("Delete PDB")
 		err = client.Delete(context.TODO(), pdb)
 		o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
wouldn't this test always get stuck deleting the machineSet during afterEach as the only machine running the  pods and suitable for it will get stuck draining the node